src/app/components-small/plugin-filter-node/plugin-filter-node.component.html

```diff
@@ -56,6 +56,7 @@
                         <mat-option value="name">Name</mat-option>
                         <mat-option value="tag">Tag</mat-option>
                         <mat-option value="version">Version</mat-option>
+                        <mat-option value="type">Type</mat-option>
```

Comment:
It should be possible to provide a [datalist](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/datalist) for the known types for if "type" is selected for this filter. This would enable basic autocomplete functions for the user while still allowing for new plugin types.

Reply:
Done. The plugin types are hard-coded:

https://github.com/UST-QuAntiL/qhana-ui/blob/0394190fdec19daaf7aa4e0d496ff1fec8092836/src/app/components-small/plugin-filter-node/plugin-filter-node.component.html#L65-L71